Podfile

```diff
@@ -19,6 +19,8 @@ target 'TomTomSDKExamples' do
   pod 'TomTomSDKMapDisplay', '0.22.1'
   pod 'TomTomSDKNavigation', '0.22.1'
   pod 'TomTomSDKNavigationEngines', '0.22.1'
+  pod 'TomTomSDKNavigationOffline', '0.22.1'
```

Comment:
Users may experience pod installation errors due to differing access permissions for offline dependencies. To avoid these issues, it's best to exclude any dependencies that are related to offline functionality.

Reply:
Actually, offline functionality is also not used at all in the example code. Will remove.